packages/runtime/src/widgets/Form/__tests__/MultiSelect.test.tsx

```diff
@@ -442,5 +436,107 @@ describe("MultiSelect Widget", () => {
       );
     });
   });
+
+  test("supports search in multiselect widget", async () => {
+    render(
+      <Form
+        children={[
+          {
+            name: "MultiSelect",
+            properties: {
+              id: "searchInput",
+              label: "Choose Option",
+              data: [
+                { label: "Option 1", value: "option1" },
+                { label: "Option 2", value: "option2" },
+                { label: "Option 3", value: "option3" },
+                { label: "Option 4", value: "option4" },
+              ],
+            },
+          },
+        ]}
+        id="form"
+      />,
+      { wrapper: FormTestWrapper },
+    );
+
+    const selector = ".ant-select-item-option-content";
+
+    userEvent.click(screen.getByRole("combobox"));
+    userEvent.type(document.querySelector("input") as HTMLElement, "Option 4");
+
+    // Wait for the combobox to reflect the selected values
+    await waitFor(() => {
+      expect(screen.getByText("Option 4", { selector })).toBeVisible();
+      expect(screen.queryByText("Option 1")).not.toBeInTheDocument();
+      expect(screen.queryByText("Option 2")).not.toBeInTheDocument();
+      expect(screen.queryByText("Option 3")).not.toBeInTheDocument();
+    });
+  });
+
+  test("supports search in multiselect widget 2", async () => {
```

Comment:
how is this different from the other test? please write a more descriptive name.